### ISSUE TITLE
Yarn fails in Windows Installation due to installed nodejs version different from package.json

### ISSUE THREAD

**Issue by colorfusion** [thepracticaldev/dev.to#8446]
**Describe the bug**

Based on the Installation guide for Windows, installing `nodejs` through `apt-get` will install the latest version (`12.18` at time of writing). But `package.json` still uses an older version of `node` (`12.16.x` at time of writing), running `bin/yarn` will generate the following error.
```sh
error dev.to@1.0.0: The engine "node" is incompatible with this module. Expected version "12.16.x". Got "12.18.0"
error Found incompatible module.
```

**To Reproduce**

1. Install nodejs based on **Installing Rails** in [Installing prequisites](https://docs.dev.to/installation/windows/#installing-prerequisites) step.
2. Observe that running `node -v` gives you a version that is different from version within `package.json`.
3. Follow through installation steps and run `bin/setup`
4. Observe error being throw when `bin/yarn` is executed.

**Expected behavior**

Expected `bin/yarn` to install packages correctly without any errors being thrown.

**Screenshots**

![image](https://user-images.githubusercontent.com/1024128/84518031-a51e7d00-ad02-11ea-8a2b-b7056cb93d66.png)

**Desktop (please complete the following information):**

- OS: Windows 10
- Browser: Firefox Browser
- Version: 76.0.1 (64-bit)

**Smartphone (please complete the following information):**

- Device:
- OS:
- Browser:
- Version:

**Additional context**

Installing `nodejs` through `nvm` instead fixes this problem, if a specific `nodejs` version must be used for the current builds.

**Comment by maestromac** [comment]
Thank you for this issue @colorfusion . We'll have to update our doc to specify nvm as a prerequisite.

do you have [nvm](https://github.com/nvm-sh/nvm#installing-and-updating) installed? once you do, you can do `nvm use` and it will use the proper version our repo specified

**Comment by colorfusion** [comment]
Yeah, managed to fix the nodejs version with `nvm`. Fortunately it works in WSL2.
Thanks a lot!

**Comment by Amorpheuz** [comment]
Not sure if this is the correct place to comment about this, but I have installed Fork to WSL2 Ubuntu completely via the Linux Guide! It fully works for me, except the test-cases (which I have yet to test) but they didn't use to work with WSL1 for me anyways. 

Will test it out and let you know if you would like to point people with WSL2 directly to the Linux installation.

**Comment by citizen428** [comment]
@Amorpheuz That's good to know! This would actually make a good post on DEV if you feel like writing one 😃

**Comment by Amorpheuz** [comment]
Great idea! Consider it done.

**Comment by nickytonline** [comment]
For reference for anyone that comes across this issue, here is the post @Amorpheuz wrote. 🔥 https://dev.to/amorpheuz/wsl2-setup-dev-to-with-linux-but-on-windows-3cdk

**Comment by Amorpheuz** [comment]
Would a installation method similar to #8186 's Fix in #8854 work for this? Such as:

Install `nvm` pre dev repo clone. Post clone, install node with version from `.nvmrc`. The Yarn installation can be performed without node as outlined in the third code block under Debian/Ubuntu on its [installation page](https://classic.yarnpkg.com/en/docs/install#debian-stable). I.e. `sudo apt update && sudo apt install --no-install-recommends yarn`

**Comment by maestromac** [comment]
@Amorpheuz yeah I like that!
